pages/plugins.vue

```diff
@@ -17,7 +17,7 @@ export default {
   },
   head() {
     return {
-      title: "hapi.dev - Plugins",
+      title: "Plugins - hapi.dev",
       head() {
         return {
           title: "hapi.js - " + this.$route.params.family,
```

Comment:
I wonder if we should not change it here too. By the way the structure is pretty odd:
```js
head() {
  return {
    head() {
       return { 
         ...
       }
    }
}
```

@jarrodyellets Do you know if that's intentional or an error slipped through?

Reply:
That looks like an error to me